src/sage/rings/polynomial/laurent_polynomial.pyx

```diff
@@ -31,8 +30,8 @@ cdef class LaurentPolynomial(CommutativeAlgebraElement):
 
         EXAMPLES::
 
-            sage: L.<x,y> = LaurentPolynomialRing(QQ) # indirect doctest
-            sage: x*y
+            sage: L.<x,y> = LaurentPolynomialRing(QQ)  # indirect doctest               # optional - sage.modules
+            sage: x*y                                                                   # optional - sage.modules
             x*y
```

Comment:
Does this (just constructing the ring and an element) depend on `sage.modules`? Then the distribution package that installs `laurent_polynomial.pyx` should install `sage.modules`. No?

Reply:
see below